src/components/SideNav/CarbonSideNav.jsx

```diff
@@ -33,7 +31,9 @@ export default class SideNav extends React.Component {
     /**
      * Required props for accessibility label on the underlying menu
      */
-    ...AriaLabelPropType,
+    /* eslint-disable */
```

Comment:
can you only enable next to the line, otherwise it'll turn off for the whole file.  Or if you do turn it off, only turn off specific rules?

Reply:
done